controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -298,6 +299,76 @@ func (r *NodeNetworkConfigurationPolicyReconciler) Reconcile(ctx context.Context
 	return ctrl.Result{}, nil
 }
 
+// The function strips unnecessary parts of the error messages so it is easier to read
```

Comment:
I am +1 on moving to a separate file, but `pkg/helper` since it's a bad package name that we will try to refactor in the futre and split it into multiple packages/files.